app/javascript/avo.base.js

```diff
@@ -49,6 +49,14 @@ function initTippy() {
 
       return title
     },
+    onShow(tooltipInstance) {
+      // Don't render tooltip if there is no content.
```

Comment:
Let me try

Reply:
It seems that initTippy is only called once when initializing tippy, rather than being invoked for each individual element. I believe tippy internally scans all the elements, iterates through them, and then applies the tooltip. We can use the `onShow` hook to prevent rendering when the content is empty.